packages/screens/DAppStore/query/util.ts

```diff
@@ -0,0 +1,8 @@
+export function getValuesFromId(option: string) {
+  const separator = "*SEPARATOR*";
```

Comment:
I see that `*SEPARATOR*` is hardcoded here and here https://github.com/TERITORI/teritori-dapp/pull/401/files#diff-4cfc8a0ab0cca2d943f4b5fbdc992248981362b4aba64ef887e1592f4e71c3d2R29. I think we should centralize this value in one place, It could be easy to make mistake in using this